spec/features/form_sections/teacher_training_data/placements/adding_trainee_placements_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Add a placement" do
+  after do
+    FormStore.clear_all(@trainee.id)
+  end
+
+  scenario "Add a new placement to an existing trainee" do
+    given_i_am_authenticated
+    and_a_trainee_exists_with_trn_received
+    and_a_school_exists
+
+    when_i_navigate_to_the_new_placement_form
+    then_i_see_the_not_found_page
+
+    when_the_feature_flag_is_active
```

Comment:
```
context "placements enabled", feature_placements: true
```

it probably better to break it out to two tests 

since it is a context set beforehand